### ISSUE TITLE
Default netstat command generates false positives

### ISSUE THREAD

**Issue by henare** [wazuh/wazuh-ansible#364]
The netstat command introduced in https://github.com/wazuh/wazuh-ansible/pull/335/files#diff-6c07bdad477b8ad14662126cf16f2713R326 generates too many false positives because it includes the PID and name of the program to which each socket belongs. This information can legitimately change, e.g. a process restarting.

Previously we had the neat use of `ss` introduced in https://github.com/wazuh/wazuh-ansible/commit/81058daf1b3baa517203417a2283eadf59831cf9. However #355 sought to realign the defaults in this repo with that of the upstream defaults so it overwrote that with this more complex netstat command. I'm not sure where the command in #335 came from though because [upstream](https://github.com/wazuh/wazuh/blob/6c709db7041356710c6780e58beb54378307694e/etc/ossec-agent.conf#L128) looks to be...

`netstat -tan |grep LISTEN |grep -v 127.0.0.1 | sort`

...which would not include the PID or program name.

I'm opening this issue rather than just a PR to revert us back to `ss` because I can understand the desire to keep us in sync with the standard defaults. Are we able to change to `ss` in the agent defaults and also do that in this repository? It seems like the tidiest configuration.

**Comment by xr09** [comment]
Hello @henare

There's an open issue about this on Wazuh core, check https://github.com/wazuh/wazuh/issues/179

The thing is we're aiming for a multi-platform command to gather open ports and the programs behind them, a bad actor could take control of your server kill Apache and start listening on port 80 for example, the program name is needed to monitor this case.

As stated on that issue there's an ongoing effort to remove this dependency and use Syscollector as a native solution.

Right now you can rewrite this value on your playbook, and the value will take precedence over the defaults so feel free to use your own choice here. Depending on your deployment you could use netstat/ss and sed/awk for refining the output.

If you're good with netstat then something like this should do: `netstat -tanp |grep LISTEN | sed 's/[0-9]*\///' | grep -v 127.0.0.1 | sort`

```bash
[root] ~# netstat -tanp |grep LISTEN | sed 's/[0-9]*\///' | grep -v 127.0.0.1 | sort
tcp        0      0 0.0.0.0:111             0.0.0.0:*               LISTEN      systemd           
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      sshd          
tcp        0      0 0.0.0.0:53              0.0.0.0:*               LISTEN      dnsmasq        
tcp6       0      0 :::111                  :::*                    LISTEN      systemd           
tcp6       0      0 :::22                   :::*                    LISTEN      sshd          
tcp6       0      0 :::53                   :::*                    LISTEN      dnsmasq   
```

**Comment by henare** [comment]
Hi @xr09, thanks very much for the detailed response :smile: 

This seems like a real can of worms :bug: I'm happy to close this in favour of it being addressed upstream in wazuh/wazuh#179.
